src/stp/Play.cpp

```diff
@@ -25,17 +25,20 @@ Status Play::update() noexcept {
         assignRoles();
     }
 
+    calculateInfoForPlay();
+
     for (auto& each : roles) {
         auto roleName{each->getName()};
         if(stpInfos.find(roleName) != stpInfos.end()) {
-            // TODO refresh robots
+            // TODO refresh views
             stpInfos.find(roleName)->second.setRobot(world->getWorld()->getRobotForId(stpInfos.find(roleName)->second.getRobot()->get()->getId()));
+            stpInfos.find(roleName)->second.setBall(world->getWorld()->getBall());
```

Comment:
This was already implemented development

Reply:
i know